src/components/AircraftPopup.jsx

```diff
@@ -1,14 +1,13 @@
-// Not in use right now
 import { observer } from 'mobx-react-lite';
 import React from 'react';
 import {
   Button, Col, Container, Row,
 } from 'react-bootstrap';
-import { DraggableCore } from 'react-draggable';
-import { Popup } from 'react-map-gl';
 
+import { isDragging } from '../draggableState';
 import { acceptFlight } from '../mqtt';
 import { aircraftStore, configurationStore, cwpStore } from '../state';
+import DraggablePopup from './DraggablePopup';
```

Comment:
I move the draggable part of the popup to a specific component to have more clean code.